lib/AArch64Parser.mly

```diff
@@ -1886,6 +1893,38 @@ instr:
 | TLBI TLBI_OP COMMA xreg
   { I_TLBI ($2, $4) }
 
+/* Pointer Authentication Code Extension */
+| PACIA1716 { I_PAC_IA (Ireg R17, Ireg R16) }
+| PACIASP { I_PAC_IA (Ireg R30, SP) }
+| PACIAZ { I_PAC_IA (Ireg R30, ZR) }
+| PACIA xreg COMMA xreg { I_PAC_IA ($2, $4) }
```

Comment:
The four rules for `PACIA`, `PACIB`, PACDA` and `PACDB` can be replaced by one:
```
| PAC xreg COMMA xreg { I_PAC ($1,$2, $4) }
```

Reply:
Of coutse, the lexer has to me modified to. This amounts to change the right hand sides of rules. Typically
```OCaml
| "PACIA" | "pacia" -> PACIA
```
is replaced by
```OCaml
| "PACIA" | "pacia" -> PAC IA
```